OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/SaveConfirmDialog.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package latexstudio.editor.menu;
+
+import java.io.File;
+import javax.swing.JOptionPane;
+import latexstudio.editor.files.FileChooserService;
+
+/**
+ * Confirm file saving and show the confirmation dialog
+ * @author WhiteHsu
+ * @param reply reply from the JOptionPane dialog
+ * @param file the selected file
+ * @param currentFIle current file saved in Editor Top Component
+ * @return the selected file object
+ */
+public class SaveConfirmDialog {
+    public static File confirmFileSave(int reply, File file, File currentFile) {
```

Comment:
I think this method could be placed in the FileChooserService. After all, it can only be used with FileChooserService anyways.